gui-tk/minsky.tcl

```diff
@@ -785,8 +797,9 @@ bind . <$meta-x> {minsky.cut}
 bind . <$meta-X> {minsky.cut}
 bind . <$meta-c> {minsky.copy}
 bind . <$meta-C> {minsky.copy}
-bind . <$meta-v> {minsky.paste}
-bind . <$meta-V> {minsky.paste}
+# Fix for ticket 1080
```

Comment:
For the future, please do not do comments like this. This code probably doesn't need commenting at all, but if necessary, comments should be brief, and understandable without  reference to anything else. If necessary, something along the lines of "see discussion on ticket #..." is OK. We may not always have access to ticket data.